server/app/services/memory_store.rb

```diff
@@ -0,0 +1,55 @@
+# Simple memory store for Rack::Attack throttling
+class MemoryStore
```

Comment:
Not a thread safe implementation. One option is to use [MonitorMixin](http://ruby-doc.org/stdlib-2.3.1/libdoc/monitor/rdoc/MonitorMixin.html) (or plain old mutex).